.github/workflows/scheduled.yaml

```diff
@@ -124,7 +124,7 @@ jobs:
           checkGlob: '**/en-US.ini'
 
       - name: Upload US English Language Files 🇺🇸
-        if: fromJSON(steps.checks.outputs.hasChangedFiles) && fromJSON(steps.nightly-checks.outputs.passed)
+        if: fromJSON(steps.nightly-checks.outputs.passed) && steps.checks.outputs.passed == 'true'
```

Comment:
If `checks` step is conditional on the `nightly-checks` step returning `true`, then isn't the first conditional superfluous?

Or does this:
```yaml
if: fromJSON(steps.nightly-checks.outputs.passed)
```
actually succeed if there is _any_ output returned by the `fromJSON` call?

Reply:
It's not necessary as the check for changed files will not yield "true" otherwise, but it makes it more explicit that both rely on this condition.

Reply:
Got it.